hydroshare/urls.py

```diff
@@ -17,6 +17,7 @@
 from hs_tracking import views as tracking
 from hs_core import views as hs_core_views
 from hs_app_timeseries import views as hs_ts_views
+from hs_app_netCDF import views as nc_views
```

Comment:
This has nothing to do with this PR and I wonder why it is here.

Reply:
Revert to the develop version.